ir/func.go

```diff
@@ -112,14 +112,18 @@ func (f *Func) Type() types.Type {
 
 // LLString returns the LLVM syntax representation of the function definition or
 // declaration.
+//
+// Function declaration.
+//
+//    'declare' Metadata=MetadataAttachment* Header=FuncHeader
+//
+// Function definition.
+//
+//    'define' Header=FuncHeader Metadata=MetadataAttachment* Body=FuncBody
```

Comment:
I move them to document since it seems is a helpful message for users.

Reply:
That's good. Would you prepare a follow-up PR that moves these for all other `LLString` methods too? So we stay consistent in documentation.

Reply:
Sounds good.

String() has similar content like LLString(), did that outdated?

Reply:
> did that outdated?

As far as I know, these comments should be up-to-date. If not, that's a bug in llir/llvm, and llir/grammar is considered the source of truth.

Reply:
For example:

```go
// String returns the LLVM syntax representation of the function as a type-value
// pair.
func (f *Func) String() string {
        return fmt.Sprintf("%s %s", f.Type(), f.Ident())
}

// LLString returns the LLVM syntax representation of the function definition or
// declaration.
```

Reply:
Feel free to add the relevant parts from the grammar, if missing. I think I may have excluded `TypeValue` because it was "simple". However, I agree that it may be better to strive for consistency.